src/app/accounts/lock.component.ts

```diff
@@ -39,5 +47,19 @@ export class LockComponent extends BaseLockComponent {
                 setTimeout(() => this.unlockBiometric(), 1000);
             }
         });
+        this.broadcasterService.subscribe(BroadcasterSubscriptionId, async (message: any) => {
```

Comment:
You need to unsubscribe from the broadcaster service onDestory. See examples in other components.

Reply:
🤦🏻 , I had missed that before, got that implemented now.